contributors/design-proposals/cluster-lifecycle/bootstrap-discovery.md

```diff
@@ -239,8 +239,8 @@ The binding of the `system:bootstrappers` (or similar) group to the ability to s
 
 ## Revision history
 
- - Initial proposal ([@jbeda](https://github.com/jbeda)): [link](https://github.com/kubernetes/community/blob/cb9f198a0763e0a7540cdcc9db912a403ab1acab/contributors/design-proposals/bootstrap-discovery.md)
- - v1.6 updates ([@jbeda](https://github.com/jbeda)): [link](https://github.com/kubernetes/community/blob/d8ce9e91b0099795318bb06c13f00d9dad41ac26/contributors/design-proposals/bootstrap-discovery.md)
+ - Initial proposal ([@jbeda](https://github.com/jbeda)): [link](https://github.com/kubernetes/community/blob/cb9f198a0763e0a7540cdcc9db912a403ab1acab/contributors/design-proposals/cluster-lifecycle/bootstrap-discovery.md)
```

Comment:
This linked to a particular commit sha. So, the earlier URL was correct. The new one isn't.